### PR TITLE
Chap 10: Updating, showing, and deleting users

### PR DESCRIPTION
**Pull request by wellcomeoh**
screen shot 2018-08-06 at 11 50 48

### CODE REVIEW

**Comment by wellcomeoh** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -21,10 +29,48 @@ def create
...
+      render :edit
+    end
+  end
+
+  def edit
+    @user = User.find_by id: params[:id]
+  end
+
+  def destroy
+    @user = User.find_by id: params[:id]
```
Comment (huongnguyenmta) [inline app/controllers/users_controller.rb]: dùng `before_action` xử lý tìm user cho các actions: show, edit, update, destroy nhé

**Comment by huongnguyenmta** [comment]
commented

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (wellcomeoh) [reply app/controllers/users_controller.rb]: dạ vâng ạ

**Comment by wellcomeoh** [comment]
ready

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -21,10 +29,50 @@ def create
...
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def edit; end
+
+  def destroy
+    @user.destroy
```
Comment (huongnguyenmta) [inline app/controllers/users_controller.rb]: check trường hợp destroy fail

**Comment by huongnguyenmta** [comment]
commented

**Comment by wellcomeoh** [comment]
ready

**Comment by huongnguyenmta** [comment]
merged
Mớt pull này đi cưng nha (h)

**Pull request merged**